### ISSUE TITLE
Potentially removing the following Raw.githack Generator section

### ISSUE THREAD

**Issue by ewhang5** [open-learning-exchange/open-learning-exchange.github.io#3268]
### Problem
The Github issues page is quite long. In addition to this, I think the original explanation of how to use Raw.githack in 5.2 is sufficient for being able to use Raw.githack. I feel that the section shown below could be slightly unnecessary.

### Steps to reproduce the problem

N/A

### Screenshots

<img width="905" alt="Screenshot 2024-05-22 at 2 48 28 PM" src="https://github.com/open-learning-exchange/open-learning-exchange.github.io/assets/59347000/90118671-425f-41ca-b947-7baef36599f4">

### Proposed solution

Remove the section or make more concise.

**Comment by xyb994** [comment]
I agree this step need quite some cleanup. I was only able to get to the earlier sections of this page. This make sense, go ahead and create a pull request for it!
